modules/extensions-support/src/test/java/org/mule/runtime/module/extension/internal/runtime/DefaultExecutionMediatorTestCase.java

```diff
@@ -101,9 +106,19 @@
 @RunWith(Parameterized.class)
 public class DefaultExecutionMediatorTestCase extends AbstractMuleContextTestCase {
```

Comment:
too many changes in this test, are we sure we're not breaking anything?

Reply:
Well, I share the concern, but on the other hand, most of the changes are a consequence of removing the ConnectionException handling from the ExceptionHandlerManager class that in some cases was completely replacing the exception tree by a new one, discarding Module and Messaging exceptions that are relevant in the context of error handling and troubleshooting (see https://github.com/mulesoft/mule/pull/9366/commits/04ccc49e9e27ef59e7d782ce868ed177ffc00c97). That particular change will allow us to make an additional enhancement regarding the error handling and the troubleshooting that consists on suppressing the ConnectionException (probably in the ModuleExceptionHandler). I'm in the process of locally running some connector tests in order to see if something breaks.